### PR TITLE
Add Full Assigned Jurisdiction Path to Patient profile

### PR DESCRIPTION
**Pull request by tstrass**
Implementation: https://github.com/SaraAlert/SaraAlert/pull/494

### CODE REVIEW

**Review by jawalonoski — COMMENTED** [review]
This looks fine and straight-forward.

What is the purpose of the new jurisdiction extension? Is it just informative? If so, String is fine.

But if it is more than that -- e.g. applications might edit the value and attempt to switch patients jurisdictions -- then maybe the string value is not such a great idea. The question then becomes, how does the web app do that today? By altering the string, or an identifier, or something else? Food for your thought.

File: fsh/patient.fsh [hunk 1]
```diff
@@ -11,7 +11,8 @@ Description: "Sara Alert outputs additional extensions on Patient resources"
   preferred-contact-time named preferred-contact-time 0..1 MS and
   symptom-onset-date named symptom-onset-date 0..1 MS and
   last-exposure-date named last-exposure-date 0..1 MS and
-  isolation named isolation 0..1 MS
+  isolation named isolation 0..1 MS and
+  full-assigned-jurisdiction-path named full-assigned-jurisdiction-path 0..1 MS and
```
Comment (ngfreiter) [inline fsh/patient.fsh]: The extra `and` at the end of the line here is making SUSHI not so happy.

File: fsh/patient.fsh [hunk 2]
```diff
@@ -53,3 +54,10 @@ Id: isolation
 Title: "Isolation"
 Description: "If the monitoree should be in the isolation workflow (omitting this extension defaults this value to false, leaving the monitoree in the exposure workflow."
 * value[x] only boolean
+
+// Full Assigned Jurisdiction Path
+Extension: FullAssignedJurisdictionPath
+Id: full-assigned-jurisdiction-path
+Title: "Full Assigned Jurisdiction Path"
+Description: "Represents the jurisdiction in which this Patient is enrolled."
```
Comment (ngfreiter) [inline fsh/patient.fsh]: You could consider having this description use "monitoree" instead of "Patient", for sake of consistency with other descriptions.

**Review by ngfreiter — CHANGES_REQUESTED** [review]
This looks good, just a typo that is causing some errors during building that needs to be fixed.

**Comment by tstrass** [comment]
> What is the purpose of the new jurisdiction extension? Is it just informative? If so, String is fine.
> 
> But if it is more than that -- e.g. applications might edit the value and attempt to switch patients jurisdictions -- then maybe the string value is not such a great idea. The question then becomes, how does the web app do that today? By altering the string, or an identifier, or something else? Food for your thought.

It's not just informative- it's there so that client apps can assign monitoree jurisdiction. As far as I can tell, this is the same way it works in the UI, although that certainly doesn't mean we need to do it that way in the API. I'd prefer something more structured than a comma separated string, though I'll have to figure out what all can go in that string (since I know it's not as simple as just "`<country>`, `<state>`, `<county>`"). @jawalonoski

**Review by ngfreiter — APPROVED** [review]

**Pull request merged**